examples/local/e2enodenetwork/main.go

```diff
@@ -235,8 +251,26 @@ func postProcessConfig(config *network.Config) {
 		},
 		"platformAllocations": []interface{}{map[string]interface{}{"amount": 4000000000000}},
 	})
+
 	camino["allocations"] = allocations
 
+	cChainGenesis["initialAdmin"] = "0x1f0e5c64afdf53175f78846f7125776e76fa8f34"
```

Comment:
Where are the provate keys of these addresses?
See caminogo/genesis/genesis_kopernikus.go where we have all keys in comments.

Reply:
Added some comments